typescript/token/contracts/HypERC20.sol

```diff
@@ -20,23 +20,18 @@ contract HypERC20 is ERC20Upgradeable, TokenRouter {
     /**
      * @notice Initializes the Hyperlane router, ERC20 metadata, and mints initial supply to deployer.
      * @param _mailbox The address of the mailbox contract.
-     * @param _interchainGasPaymaster The address of the interchain gas paymaster contract.
      * @param _totalSupply The initial supply of the token.
      * @param _name The name of the token.
      * @param _symbol The symbol of the token.
      */
     function initialize(
         address _mailbox,
-        address _interchainGasPaymaster,
         uint256 _totalSupply,
         string memory _name,
         string memory _symbol
     ) external initializer {
-        // transfers ownership to `msg.sender`
-        __HyperlaneConnectionClient_initialize(
-            _mailbox,
-            _interchainGasPaymaster
-        );
+        // initialize router
```

Comment:
Just flagging that this is sorta a regression for warp routes because now you can't configure your own IGP to use with warp routes (unless we're planning on having the default hook be a configuration based one?)

Reply:
indeed, in hindsight I probably wouldn't have removed this but still possible to configure subsequently via router setHook